tasks/main.yml

```diff
@@ -23,3 +39,17 @@
       # See: https://github.com/ansible/ansible-lint/issues/1894
       command: "{{ nix_script_dir }}/install_nix.sh"  # noqa no-changed-when
   when: nix_directory is changed  # noqa no-handler
+
+- name: Set up nix flakes
+  block:
+    - name: Install nix flakes package
+      become: true
+      become_flags: "--login"
```

Comment:
Why is this necessary?

Reply:
we need `nix` to be available on the path and this wasn't happening in my original version.

One option is to manually invoke the script added to the user profile, but it's a bit ugly

I can check this is still needed though actually

Reply:
Yes: removing it fails with `"/bin/sh: 1: nix-env: not found"`

Reply:
Great, thanks for the explanation.